modules/claims_api/app/controllers/claims_api/v2/application_controller.rb

```diff
@@ -115,6 +119,40 @@ def user_represents_veteran?
 
         rep.poa_codes.include?(veteran_poa_code)
       end
+
+      private
+
+      def validate_ccg_token!
```

Comment:
seems like overkill to abstract/consolidate with a single duplication, but just noting that this is a duplication of the same `validate_ccg_token` method in the v1 `ApplicationController`